csfieldguide/chapters/content/en/complexity-and-tractability/sections/tractability.md

```diff
@@ -37,7 +37,7 @@ It may be more meaningful to students it this is converted to some geological ti
 
 {panel end}
 
-{button-link link="http://www.csfieldguide.org.nz/releases/1.9.9/_static/widgets/tract-scaling-v2.html" text="View big time calculator"}
+{interactive slug="algorithm-timer" type="whole-page"}
```

Comment:
Perhaps remove the 2 decimal places if they aren't needed, like for a number such as 10000000000000.00

Reply:
Also how about adding order nlog(n), since that is also very common?